### PR TITLE
Version updates

### PR DESCRIPTION
**Pull request by hendrikebbers**
Updated some dependencies

---
This change is [Reviewable](https://reviewable.io/reviews/canoo/dolphin-platform/823)

### CODE REVIEW

File: platform/dolphin-platform-distributed-eventbus/src/test/java/com/canoo/dp/impl/server/event/EventStreamSerializerTests.java [hunk 1]
```diff
@@ -397,12 +408,18 @@ public String readLine() throws IOException {
...
+        }
     }
 
     private class ByteObjectDataOutput implements ObjectDataOutput {
 
         private final StringBuffer content = new StringBuffer();
 
+
         @Override
         public void writeByteArray(byte[] bytes) throws IOException {
```
Comment (giftkugel) [inline platform/dolphin-platform-distributed-eventbus/src/test/java/com/canoo/dp/impl/server/event/EventStreamSerializerTests.java]: Add `final` keyword.

File: platform/dolphin-platform-distributed-eventbus/src/test/java/com/canoo/dp/impl/server/event/EventStreamSerializerTests.java [hunk 2]
```diff
@@ -537,6 +559,11 @@ public void writeChars(String s) throws IOException {
         public void writeUTF(String s) throws IOException {
```
Comment (giftkugel) [inline platform/dolphin-platform-distributed-eventbus/src/test/java/com/canoo/dp/impl/server/event/EventStreamSerializerTests.java]: Add `final` keyword.

File: platform/dolphin-platform-distributed-eventbus/src/test/java/com/canoo/dp/impl/server/event/EventStreamSerializerTests.java (hunk 1 above)
Comment (hendrikebbers) [reply platform/dolphin-platform-distributed-eventbus/src/test/java/com/canoo/dp/impl/server/event/EventStreamSerializerTests.java]: done

File: platform/dolphin-platform-distributed-eventbus/src/test/java/com/canoo/dp/impl/server/event/EventStreamSerializerTests.java (hunk 2 above)
Comment (hendrikebbers) [reply platform/dolphin-platform-distributed-eventbus/src/test/java/com/canoo/dp/impl/server/event/EventStreamSerializerTests.java]: done

**Pull request merged**